tests/test_applets/featureSelection/testNewFeatureSelection.py

```diff
@@ -16,9 +17,10 @@
 logger = logging.getLogger(__name__)
 
 
-class TestCompareOpFeatureSelectionToOld():
-    @classmethod
-    def setupClass(cls):
+
+class TestCompareOpFeatureSelectionToOld:
```

Comment:
Purpose of this test is unclear. It compares following two classes:
```
from ilastik.applets.featureSelection.opFeatureSelection import OpFeatureSelection
from lazyflow.operators.oldVigraOperators import OpFeatureSelection as OpFeatureSelectionOld
```
But they are literally the same. They have minor difference. The "new" one looks like small code cleanup of copy pasted old. Maybe we should remove this test and oldVigraOperators file or just a single class.

Reply:
This test is in place to ensure that any changes to the new code, don't change the basic functionality. @FynnBe's cleanup/rewrite was very important in making the code there more maintainable. And just because it's such an important part of the pipeline, it was important to make sure that functionality doesn't change. It does fail sometimes, yes, maybe we should rather make the comparison more robust.